src/Fixer/Phpdoc/PhpdocAlignFixer.php

```diff
@@ -245,7 +245,7 @@ protected function createConfigurationDefinition(): FixerConfigurationResolverIn
             'spacing',
             'Spacing between tag, hint, comment, signature, etc. You can set same spacing for all tags using a positive integer or different spacings for different tags using an associative array of positive integers `[\'tagA\' => spacingForA, \'tagB\' => spacingForB]`. If you want to define default spacing to more than 1 space use `_default` key in config array, e.g.: `[\'tagA\' => spacingForA, \'tagB\' => spacingForB, \'_default\' => spacingForAllOthers]`.'
         );
-        $spacing->setAllowedTypes(['int', 'int[]'])
+        $spacing->setAllowedTypes(['int', 'array']) // arrray{tag: int}
```

Comment:
```suggestion
        $spacing->setAllowedTypes(['int', 'array']) // array<string, int>
```

`array{tag: int}` is a shape with one possible key.